lib/index.js

```diff
@@ -165,9 +165,54 @@ export default Adapter => class PostgreSQLAdapter extends Adapter {
 
     for (let field in options.match) {
       const value = options.match[field]
-      const isArray = fields[field][isArrayKey]
+      const isArray = (fields[field] ?
+              fields[field][isArrayKey] :
+              fields[field.split('_')[0]][isArrayKey])
```

Comment:
To be future proof, when you want to do custom filtering on arrays. But can for now be replaced by just returning false.